frontend/package.json

```diff
@@ -47,7 +47,9 @@
 		]
 	},
 	"devDependencies": {
-		"prettier": "2.4.1"
+		"@material-ui/core": "^4.12.3",
```

Comment:
material ui should be a normal dependency, not dev!

Reply:
I think IntelliJ did that...